.gitignore

```diff
@@ -0,0 +1,3 @@
+# macOS
+.DS_Store
+
```

Comment:
Redundant? Maybe IDE specific?

Reply:
This is for macOS and unrelated to an IDE. macOS creates this file when you visit a folder in Finder. It is needed to avoid adding it inadvertently.

Reply:
I want to raise the bar and see if we can add all these:
```
Thumbs.db
.DS_Store

.idea/
*.iml

__pycache__
.pytest_cache

.coverage
coverage.xml

.gradle/
.settings/
out.txt
```

Reply:
This is the standard one we use in opensearch-build repo.

Reply:
Agreed. I'll add the above.

Reply:
If we really want to 'go wild' then there is [this](https://github.com/github/gitignore).

Reply:
I didn't know MacOS added those default, sorry!